src/tensors/braidingtensor.jl

```diff
@@ -101,6 +107,11 @@ function Base.copy!(t::TensorMap, b::BraidingTensor)
     space(t) == space(b) || throw(SectorMismatch())
     fill!(t, zero(eltype(t)))
     for (f1, f2) in fusiontrees(t)
+        if f1 == nothing || f2 == nothing
+            copy!(t.data,one(t.data))
```

Comment:
can you replace this line with `_one!(t.data)` to avoid allocating a `one(t.data)` first.